service/chartconfig/v1/helm/helm.go

```diff
@@ -122,7 +122,11 @@ func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.Insta
 
 // UpdateReleaseFromTarball updates the given release using the chart packaged
 // in the tarball.
-// TODO
 func (c *Client) UpdateReleaseFromTarball(releaseName, path string, options ...helmclient.UpdateOption) error {
```

Comment:
Does it make sense to have a test for this in `helm_test.go`?

Reply:
Absolutely, added :)